src/patients/util/is-possible-duplicate-patient.ts

```diff
@@ -0,0 +1,13 @@
+import Patient from '../../shared/model/Patient'
+
+export function isPossibleDuplicatePatient(newPatient: Patient, existingPatient: Patient) {
```

Comment:
I would expect a function named `isPossibleDuplicatePatient` to return a boolean rather than a object. 

I think this function can be simplified as: 

```ts
export function isPossibleDuplicatePatient(newPatient: Patient, existingPatient: Patient) {
	return newPatient.givenName === existingPatient.givenName &&
    newPatient.familyName === existingPatient.familyName &&
    newPatient.sex === existingPatient.sex &&
    newPatient.dateOfBirth === existingPatient.dateOfBirth
}

```